crane-cloud/frontend
# add annonymous users and implementation of project invitations

# Description

This Pr shows the registered and unregistered users on project.
It also adds the feature of accepting or declining collaboration on a project

## Type of change



## Trello Ticket ID

https://trello.com/c/e4KJykvH
https://trello.com/c/ecQf4ZFz

## How Can This Been Tested?

Run this branch and check out project settings, you can invite an anonymous user, see invitation statuses and in the case that you are invited to a project. you can either denay or accept

## Checklist:


# Screenshots
![image](https://user-images.githubusercontent.com/69305400/203910788-217f9853-e895-423b-a5e1-47826bee2241.png)
![image](https://user-images.githubusercontent.com/69305400/203911239-d5a8ac82-ca4e-4007-80e1-a07ed89fd9f8.png)
![image](https://user-images.githubusercontent.com/69305400/203911352-c7e4fb16-979c-4a96-972b-2837e05c58c0.png)
